lib/interface/cli/helpers/general.js

```diff
@@ -122,13 +143,18 @@ const crudFilenameOption = (yargs, options = {}) => {
         .coerce(options.name || 'filename', (arg) => {
             try {
                 const rawFile = fs.readFileSync(path.resolve(process.cwd(), arg), 'utf8');
+                let content;
                 if (arg.endsWith('.json')) {
-                    return options.raw ? rawFile : JSON.parse(rawFile);
+                    content = options.raw ? rawFile : JSON.parse(rawFile);
```

Comment:
since you removed the `return` statements, you need to use `if else` instead of just `if`

Reply:
but just for my understanding there cant be a file that ends with both of this, so it's just for sack of more correct code writing?